NuRadioMC/utilities/cross_sections.py

```diff
@@ -92,7 +92,7 @@ def csms(energy, inttype, flavors):
     if type(inttype) == str:
         inttype = np.array([inttype] * energy.shape[0])
 
-    if (type(flavors) == int or type(flavors) == np.int64):
+    if isinstance(flavors, (int, np.integer)):
```

Comment:
That actually is the wrong direction. 
I think it should be no.int_ if we really want the int64 here. 
https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.int_